Dockerfile

```
@@ -0,0 +1,21 @@
+FROM ubuntu/mysql:8.0-20.04_beta
+
+ENV CONFIG_VERSION="0.8.22-1"
+
+RUN set -eux && \
+	apt-get update && \
+	apt-get install -y wget lsb-release gnupg
+
+RUN set -eux && \
```

```suggestion
RUN set -eux && export DEBIAN_FRONTEND=noninteractive \
```

If you export this variable upfront, you can remove it from each of the lines below and increase readability a fair bit.

---

updated in [69ec1f1](https://github.com/canonical/mysql-container/pull/1/commits/69ec1f179f718412815e44d05e7e846f96774445)